setup.py

```diff
@@ -30,14 +40,24 @@ def get_version():
 URL = 'http://nilearn.github.com'
 LICENSE = 'new BSD'
 DOWNLOAD_URL = 'http://nilearn.github.com'
-VERSION = get_version()
+VERSION = _VERSION_GLOBALS['__version__']
+
 
 if __name__ == "__main__":
+    if is_installing():
+        _VERSION_GLOBALS['_check_module_dependencies'](manual_install_only=True)#eval('_check_module_dependencies(manual_install_only=True)', _VERSION_GLOBALS)
+
     old_path = os.getcwd()
```

Comment:
I am guessing this four lines can now be removed now that you do the chdir above?